specification/healthdataaiservices/HealthDataAIServices.Management/client.tsp

```diff
@@ -7,3 +7,25 @@ using Azure.ClientGenerator.Core;
   "HealthDataAIServicesMgmt",
   "python"
 );
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Using the AI acronym similar to what was done for OpenAI"
+namespace Microsoft.HealthDataAIServices {
+  @@access(Operations.list, Access.internal, "csharp");
+
+  @@clientName(PrivateEndpointConnectionResource,
+    "HealthDataAIServicePrivateEndpointConnectionResource",
+    "csharp"
+  );
+  @@clientName(PrivateLinkResource,
+    "HealthDataAIServicePrivateLinkResourceData",
```

Comment:
I believe this is not needed anymore since we fixed it in the generator https://github.com/Azure/autorest.csharp/pull/4938.

Reply:
let me try if we could remove this